website/docs/d/schema_template_anp_epg_contract.html.markdown

```diff
@@ -26,16 +26,14 @@ data "mso_schema_template_anp_epg_contract" "contract" {
 
 ## Argument Reference ##
 
-* `schema_id` - (Required) SchemaID of Application Network Profile(ANP) EPG.
-* `template_name` - (Required) Template name of ANP EPG .
-* `anp_name` - (Required) Name of the Application Network Profile.
-* `epg_name` - (Required) Name of the Endpoint Group.
-* `relationship_type` - (Optional) Relationship Type of the ANP EPG Contract on MSO UI.
-* `contract_name` - (Required) Name of the contract.
-
-
+* `schema_id` - (Required) The schema ID of the EPG.
+* `template_name` - (Required) The template name of the EPG.
+* `anp_name` - (Required) The name of the ANP.
+* `epg_name` - (Required) The name of the EPG.
+* `contract_name` - (Required) The name of the Contract.
+* `contract_schema_id` - (Optional) The schema ID of the Contract. The `schema_id` of the EPG. will be used if not provided. 
+* `contract_template_name` - (Optional) The template name of the Contract. The `contract_template_name` of the EPG. will be used if not provided. 
```

Comment:
should it be Optional -> Read-only ?

Reply:
no since this is possible input on datasource